lua/entities/gmod_wire_expression2/core/custom/cfc_acf.lua

```diff
@@ -0,0 +1,27 @@
+CFCE2Lib.RegisterExtension( "cfc_e2_lib", true )
+
+
+e2function void entity:acfAdminDisable()
+    if not IsValid( this ) then return end
+    if not self.player:IsAdmin() then return end
+    
+    this.cfcE2LibACFDisableDamage = true 
+    
+end
+
+e2function void entity:acfAdminEnable()
+    if not IsValid( this ) then return end
+    if not self.player:IsAdmin() then return end
+    
+    this.cfcE2LibACFDisableDamage = false
+    
+end
+
+local function acfDisabledCheck(_, Entity, _, _, _, _, _, _)
+    if not IsValid( Entity ) then return end
+    
+    return not Entity.cfcE2LibACFDisableDamage
+    
+end

```

Comment:
You'll only want to return `false` here to prevent the damage.

```suggestion
local function acfDisabledCheck(_, Entity, _, _, _, _, _, _)
    if not IsValid( Entity ) then return end
    if Entity.cfcE2LibACFDisableDamage then return false end
end
```